setup.py

```diff
@@ -13,13 +13,20 @@
 
 from setuptools import setup, find_packages
 import  glob
-
+import re
 
 
 f = open("./requirements.txt",'r')
 install_req=f.readlines()
 f.close()
 
+def reformat_git_req(req_str):
```

Comment:
I would be just easier to stop assuming requirements.txt and install_requires are the same. Even if it means some duplication.

Reply:
tests_require is deprecated. But now extras_require serves its purpose.
I will try to use it then. If it will serve the purpose, it looks better, of course.

Reply:
oh good. We will upload next released version to pypi, but for now testing can rely on master. Maybe even fixed commit of it.